README.md

```diff
@@ -36,6 +36,7 @@ Usage: om-wikiparser <COMMAND>
 Commands:
   get-articles  Extract, filter, and simplify article HTML from Wikipedia Enterprise HTML dumps
   get-tags      Extract wikidata/wikipedia tags from an OpenStreetMap PBF dump
+  tag-errors    Write errors found while parsing an OSM tag file to stdout in TSV format
```

Comment:
Should it be run after get-articles or get-tags? Does it make sense to clarify usage cases here?

Reply:
It takes the output of `get-tags` or `osmconvert`, I'll clarify that.

Reply:
It would be great to see the main usage cases in the "how to prepare data for OM generator" context documented either in readme, or in the help output.